neat/tools/fittools/compartmentfitter.py

```diff
@@ -684,12 +689,12 @@ def createTreeSOV(self, use_all_channels=False, eps=1.):
         """
         # create new tree and empty channel storage
         tree = self.tree.__copy__(new_tree=FitTreeSOV())
-        if use_all_channels:
+        if self.use_all_channels_for_passive:
```

Comment:
the initial value `None` would be interpreted as `False` here. why not initialize it to `False`?

Reply:
choosing `True` as default is more in line with the paper